tmt/steps/report/reportportal.py

```diff
@@ -408,22 +403,17 @@ def get_defect_type_locator(self, session: requests.Session,
         return str(dt_locator)
 
     def rp_api_get(self, session: requests.Session, path: str) -> requests.Response:
-        response = session.get(url=f"{self.url}/{path}",
-                               headers=self.headers)
+        response = session.get(url=f"{self.url}/{path}", headers=self.headers)
         self.handle_response(response)
         return response
 
     def rp_api_post(self, session: requests.Session, path: str, json: JSON) -> requests.Response:
-        response = session.post(url=f"{self.url}/{path}",
-                                headers=self.headers,
-                                json=json)
+        response = session.post(url=f"{self.url}/{path}", headers=self.headers, json=json)
         self.handle_response(response)
         return response
 
     def rp_api_put(self, session: requests.Session, path: str, json: JSON) -> requests.Response:
-        response = session.put(url=f"{self.url}/{path}",
-                               headers=self.headers,
-                               json=json)
+        response = session.put(url=f"{self.url}/{path}", headers=self.headers, json=json)
```

Comment:
Several unrelated changes, just reformatting the source code. Is that doing of pre-commit, can it be undone?

Reply:
It was done by my editor and I commited it. Reverted: https://github.com/teemtee/tmt/pull/3331/commits/24c266a5288b0b59e765084a12697174e496511b